src/seedu/addressbook/data/person/Address.java

```diff
@@ -6,14 +6,14 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public class Address extends Contact {
 
     public static final String EXAMPLE = "123, some street";
     public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
 
-    public final String value;
-    private boolean isPrivate;
+  //  public final String value;
```

Comment:
remove unnecessary code completely; you can always recover it from previous versions RCS \#FTW